examples/http/server.cc

```diff
@@ -0,0 +1,71 @@
+#include "server.hpp"
+#include "tracer_common.hpp"
+
+#include <iostream>
+#include <thread>
+
+namespace
+{
+class RequestHandler : public HTTP_SERVER_NS::HttpRequestCallback
+{
+public:
+  virtual int onHttpRequest(HTTP_SERVER_NS::HttpRequest const &request,
+                            HTTP_SERVER_NS::HttpResponse &response) override
+  {
+    opentelemetry::trace::StartSpanOptions options;
+    options.kind = opentelemetry::trace::SpanKind::kServer;  // server
+
+    auto span =
+        get_tracer("http-server")
+            ->StartSpan(
+                "handle_request",
+                {{"req_url", request.uri}, {"req_method", request.method}, {"component", "http"}},
+                options);
+    auto scope = get_tracer("http_server")->WithActiveSpan(span);
+    for (auto &kv : request.headers)
+    {
+      span->SetAttribute("http.req.header" + std::string(kv.first.data()), kv.second);
+    }
+    if (request.uri == "/helloworld")
+    {
+      span->AddEvent("Processing request");
+      response.headers["Content-Type"] = "text/plain";
```

Comment:
nit: it'd be good to add a few standard mime types as constants in the HTTP server header.

Reply:
Thanks. We already have few mime types already there in http server lib, have used them now.